src/client/cli/cmd/transfer.cpp

```diff
@@ -247,15 +252,15 @@ mp::ParseCode cmd::Transfer::parse_destination(mp::ArgParser* parser, bool allow
         {
             if (!destination.isWritable())
```

Comment:
Relevance here might be limited, but the `destination` var hides the member of the same name, making the code confusing. Think this might be a good chance to fix that? :yum: (this really refers to line 249, but github wouldn't let me comment there)

Reply:
Yep, good catch!  Renamed the local var.